tests/ui/js/testUtils.js

```diff
@@ -114,10 +114,20 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         return element.selectorName || element.selector || element.toString();
     };
 
-    sjrk.storyTelling.testUtils.checkFirstBlockCheckbox = function (blockManager) {
+    sjrk.storyTelling.testUtils.checkBlockCheckboxes = function (blockManager, options) {
```

Comment:
It seems the only option that `options` object accepts is `checkFirstBlock`. Would it be more straightforward and efficient to pass in `checkFirstBlock` as the 2nd argument rather than passing in an object?

Reply:
I think this is a case of adding a more flexible structure ahead of time for something I can't foresee. I don't need it for now, if ever, so I'll remove it :)